src/compass/utils/lut.py

```diff
@@ -23,7 +30,19 @@ def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
     az_bistatic = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
     rg_doppler = burst.geometrical_and_steering_doppler(range_step=rg_step, az_step=az_step)
 
+    if path_dem is None:
+        # Skip the FM mismatch rate calculation
+        az_fm_mismatch = None
+    else:
+        az_fm_mismatch = burst.az_fm_rate_mismatch_mitigation(path_dem,
+                                                              path_scratch,
+                                                              rg_step=rg_step,
+                                                              az_step=az_step)
+
```

Comment:
I don't like the redundant rename of the variables below. Can we do the following to at the beginning of this function:
```python
    az_lut = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
    rg_lut = burst.geometrical_and_steering_doppler(range_step=rg_step, az_step=az_step)
```